mapper/src/main/java/org/jboss/pnc/mapper/UrlTreeSetAndOperationParametersConverter.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014-2022 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.pnc.mapper;
+
+import javax.enterprise.context.ApplicationScoped;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+import java.util.stream.Collectors;
+
+/**
+ * Converts between operation parameters, which represent URL and list of URLs.
+ */
+@ApplicationScoped
+public class UrlTreeSetAndOperationParametersConverter {
```

Comment:
TreeSet? it'S list now

Reply:
[c364881](https://github.com/project-ncl/pnc/pull/3816/commits/c364881eadcf9c934400e83e12d9f391b2a5103b)